src/scripts/components/main/data-viewer/data-viewer.tsx

```diff
@@ -87,59 +95,79 @@ const Globes: FunctionComponent<Props> = ({backgroundColor, markers = []}) => {
     setCurrentView(globalGlobeView);
   }, [globalGlobeView]);
 
-  return (
-    <div className={styles.globes}>
-      {mainLayerDetails && (
-        <LayerLegend
-          id={mainLayerDetails.id}
-          values={[mainLayerDetails.maxValue, mainLayerDetails.minValue]}
-          unit={mainLayerDetails.units}
-        />
-      )}
-      {compareLayerDetails && (
-        <LayerLegend
-          id={compareLayerDetails.id}
-          values={[compareLayerDetails.maxValue, compareLayerDetails.minValue]}
-          unit={compareLayerDetails.units}
-          isCompare={true}
-        />
-      )}
+  const showGlobeNavigation =
+    (!mainLayerDetails && !compareLayerDetails) ||
+    [mainLayerDetails, compareLayerDetails].some(
+      layer => layer && layer.type !== LayerType.Gallery
+    );
+
+  const getDataWidget = ({
+    imageLayer,
+    layerDetails,
+    active,
+    action
+  }: {
+    imageLayer: GlobeImageLayerData | null;
+    layerDetails: Layer | null;
+    active: boolean;
+    action: () => void;
+  }) => {
+    if (imageLayer?.type === LayerType.Gallery) {
+      return <Gallery imageLayer={imageLayer} />;
+    }
+
+    return (
       <Globe
         markers={markers}
         backgroundColor={backgroundColor}
-        active={isMainActive}
+        active={active}
         view={currentView}
         projectionState={projectionState}
-        imageLayer={mainImageLayer}
-        basemap={mainLayerDetails?.basemap || null}
+        imageLayer={imageLayer}
+        basemap={layerDetails?.basemap || null}
         spinning={globeSpinning}
         flyTo={flyTo}
-        onMouseEnter={() => setIsMainActive(true)}
-        onTouchStart={() => setIsMainActive(true)}
+        onMouseEnter={action}
```

Comment:
I guess this action can be hardcoded because it has to be this one for the globe and it saves one parameter :)

Reply:
Where would you hardcode it? Its `() => setIsMainActive(true)` for the main globe and `() => setIsMainActive(false)` for the compare globe.

Reply:
I wouldn't make it a parameter of the `getDataWidget` function and write in line 130 `onMouseEnter={() => setIsMainActive(true)}`

Reply:
But then it would be the same action for both globes. At the moment one sets it `true` and the other `false`

Reply:
Ah i got this wrong :) Makes sense!